src/PKSim.Presentation/Presenters/Charts/EditTimeProfileAnalysisChartPresenter.cs

```diff
@@ -183,7 +188,13 @@ private void showAnalysis()
       private void calculatePKAnalysis()
       {
          var chartData = CreateChartData();
-         _pkAnalysisPresenter.CalculatePKAnalysis(PopulationDataCollector, chartData);
+         _pkAnalysisPresenter.CalculatePKAnalysisOnCurves(PopulationDataCollector, chartData);
+
+         var fields = PopulationAnalysisChart.PopulationAnalysis.AllFields.OfType<PopulationAnalysisOutputField>().Select(x => x.QuantityPath);
+         var pks = fields.SelectMany(x => ((PopulationSimulation)PopulationDataCollector).PKAnalyses.AllPKParametersFor(x));
```

Comment:
why do you kno where that this is a PopulationSimulation? This  cast is suspicious but I am not sure

Reply:
I think it's used in population comparison. This would crash